src/Raven.Studio/typescript/components/pages/database/tasks/ongoingTasks/OngoingTasksPage.tsx

```diff
@@ -436,291 +436,291 @@ export function OngoingTasksPage() {
                 )}
             </StickyHeader>
             <Row className="gy-sm">
-            <div className="flex-vertical">
```

Comment:
revert changes in this file?

Reply:
Added that specifically to make the build green - removed :-)